qemu/tests/iperf_test.py

```diff
@@ -155,6 +171,9 @@ def rss_test():
     c_info.extend([c_options, None])
 
     try:
+        if params.get_boolean("rss_test"):
+            rss_test()
```

Comment:
So the function is to check if RSS is supported before the iperf testing, suggest rename to rss_check.

Reply:
Done, thanks.